nautobot/core/templates/inc/nav_favorites.html

```diff
@@ -0,0 +1,31 @@
+<ul class="list-unstyled">
+    <li class="sidenav-link-group text-center{% if request.user.navbar_favorites %} d-none{% endif %}"
+        style="text-transform: none;" id="favorites-empty">
+        You don't have any favorite pages yet.
+    </li>
+
+    {% for item in request.user.navbar_favorites %}
+        {% if item.link and item.name %}
+            <li class="sidenav-link-wrapper" id="sidenav-favorite-link-{{ forloop.counter }}">
+                <a class="sidenav-link flat vstack align-items-start" href="{{ item.link }}">
+                    {{ item.name }}
+                    {% if item.tab_name and item.group_name %}
+                        <small class="text-secondary">{{ item.tab_name }} / {{ item.group_name }}</small>
+                    {% endif %}
+                </a>
+                <button class="sidenav-favorite active"
+                        type="button"
+                        hx-post="{% url "user:navbar_favorites" %}"
+                        hx-vals='{"link": "{{ item.link }}", "name": "{{ item.name }}", "tab_name": "{{ item.tab_name }}", "group_name": "{{ item.group_name }}", "remove": true}'
```

Comment:
Are all of these necessary? I would have thought just `link` and `remove: true` would suffice.